templates/single.html

```diff
@@ -1,6 +1,6 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
-<!-- wp:group {"tagName":"main","align":"full","layout":{"type":"constrained"}} -->
+<!-- wp:group {"tagName":"main","align":"full"} -->
 <main class="wp-block-group alignfull"><!-- wp:group {"style":{"spacing":{"padding":{"top":"var:preset|spacing|50"},"margin":{"bottom":"var:preset|spacing|40"}}},"layout":{"type":"constrained"}} -->
```

Comment:
```html
<!-- wp:group {"tagName":"main","align":"full"} -->
<main class="wp-block-group alignfull">
```

can be further improved to 

```html
<!-- wp:group {"tagName":"main"} -->
<main class="wp-block-group">
```

Reply:
Have you tested if this change still allows for full width blocks inside the post content?

Reply:
Yes. It does work on WP 6.4-beta2 with Gutenberg 16.7.1.

The parent here is `.wp-site-blocks` which has no layout styles. Full/wide alignment only works within a constrained layout container, so having it here is redundant and editor does not even display controls to set/change the alignment.